runtime/compiler/optimizer/J9RecognizedCallTransformer.cpp

```diff
@@ -402,29 +405,50 @@ iflcmpeq -> <Block_F>
 end Block_C
 
 start Block_D
-astore <object>
+astore <object-2>
   aloadi ramStaticsFromClass
-   ...
+    aloadi  <classFromJavaLangClass>
+      aload  <object-1>
```

Comment:
You may need to show a store `object-1` so that it is clear where it came from and what relation it has to the original unsafe call

Reply:
I'll update the comments as below: 

```
start Block_A
  ...
xcall Unsafe.xxx
  ...
  <load of unsafe call argX>
  ...
end Block_A

Blocks after the transformation: ===>

start Block_A
...
 astore  object-1
   <load of unsafe call argX>
...
ifacmpeq -> <Block_E>
  object-1
  aconst null
end Block_A
```

Reply:
Addressed in 8754e4b